src/DonationForms/Blocks/DonationFormBlock/Controllers/BlockRenderController.php

```diff
@@ -38,12 +38,13 @@ public function render(array $attributes)
         $embedId = $blockAttributes->blockId ?? '';
 
         $viewUrl = $this->getViewUrl($donationForm, $embedId);
+        $formUrl = add_query_arg(['p' => $blockAttributes->formId], site_url('?post_type=give_forms'));
 
         /**
          * Note: iframe-resizer uses querySelectorAll so using a data attribute makes the most sense to target.
          * It will also generate a dynamic ID - so when we have multiple embeds on a page there will be no conflict.
          */
-        return "<div class='root-data-givewp-embed' data-src='$viewUrl' data-givewp-embed-id='$embedId' data-form-format='$blockAttributes->formFormat' data-open-form-button='$blockAttributes->openFormButton'></div>";
+        return "<div class='root-data-givewp-embed' data-form-url='$formUrl' data-src='$viewUrl' data-givewp-embed-id='$embedId' data-form-format='$blockAttributes->formFormat' data-open-form-button='$blockAttributes->openFormButton'></div>";
```

Comment:
Can we actually use `esc_url()` for the data-attribute?  Technically `site_url` can be filtered so better to be safe here.

Reply:
Resolved https://github.com/impress-org/givewp/pull/7080/commits/f97f48ca37a997188cb6989f06d8dddc4dc2f79a